docs/cedarling/cedarling-multi-issuer.md

```diff
@@ -0,0 +1,769 @@
+      "token_metadata": {
+        "id_token": {
+          "entity_type_name": "Jans::Id_Token",
+          "token_id": "jti"
+        }
+      }
+    },
+    "custom_service_issuer": {
+      "name": "CustomService",
+      "description": "Custom Service Provider",
+      "openid_configuration_endpoint": "https://service.example.com/.well-known/openid-configuration",
+      "token_metadata": {
+        "custom_token": {
+          "entity_type_name": "Custom::ServiceToken",
+          "token_id": "jti"
+        }
+      }
+    }
+  }
+}
+```
+
+### Cedar Schema for Multi-Issuer Tokens
+
+Define token entity schemas for enhanced type safety:
+
+```cedar
+namespace Jans {
+  entity Access_Token = {
+    "token_type": String,
+    "jti": String,
+    "issuer": String,
+    "exp": Long,
+    "validated_at": Long,
+    "scope": Set<String>,
+    "aud": Set<String>,
+    "client_id": String
+  } tags String;
+  
+  entity Id_Token = {
+    "token_type": String,
+    "jti": String,
+    "issuer": String,
+    "exp": Long,
+    "validated_at": Long,
+    "sub": String,
+    "email": String,
+    "email_verified": Boolean
+  } tags String;
+}
+
+namespace Custom {
+  entity ServiceToken = {
+    "token_type": String,
+    "jti": String,
+    "issuer": String,
+    "exp": Long,
+    "validated_at": Long,
+    "service_id": String,
+    "permissions": Set<String>
+  } tags String;
+}
+```
+
+## Error Handling
+
+### Token Validation Failures
+
+```python
+# Individual token validation failures are handled gracefully
+tokens = [
+    TokenInput(mapping="Jans::Access_Token", payload="valid_token"),
+    TokenInput(mapping="Jans::Id_Token", payload="invalid_token"),  # Will be ignored
+    TokenInput(mapping="Acme::CustomToken", payload="valid_custom_token")
+]
+
+# Authorization continues with valid tokens
+# Invalid tokens are logged but don't block processing
+result = cedarling.authorize_multi_issuer(request)
+```
+
+### Non-Deterministic Tokens
+
+```python
+# ERROR: Multiple tokens of same type from same issuer
+tokens = [
+    TokenInput(mapping="Jans::Access_Token", payload="token1"),  # From Acme
+    TokenInput(mapping="Jans::Access_Token", payload="token2"),  # Also from Acme - ERROR!
+]
+
+# This will fail because it's non-deterministic
+# Which token should policies reference?
+```
+
+### Trusted Issuer Validation
+
+```python
+# Tokens from unknown issuers are rejected
+tokens = [
+    TokenInput(
+        mapping="Jans::Access_Token",
+        payload="token_from_unknown_issuer"  # ERROR if issuer not in trusted issuers
+    )
+]
+
+# Only tokens from issuers configured in policy store are accepted
+```
+
+## Best Practices
+
+### 1. Use Descriptive Issuer Names
+
+Configure clear, predictable issuer names in your policy store:
+
+```json
+{
+  "name": "AcmeCorp",  // Good - clear and predictable
+  "name": "Issuer1"    // Bad - unclear what this represents
+}
+```
+
+### 2. Start Schema-Less for Development
```

Comment:
List did not render correctly
<img width="807" height="174" alt="image" src="https://github.com/user-attachments/assets/857558c6-ca25-4300-be0d-7b79a3f54452" />